Gemfile.lock

```diff
@@ -207,7 +207,7 @@ DEPENDENCIES
   tzinfo-data
 
 RUBY VERSION
-   ruby 3.1.3p185
+   ruby 3.1.2p20
```

Comment:
Kindly remove these changes made in Gemfile.lock from the PR, it will create a conflict. You can do so with the following command:
```
$ git checkout model
$ git checkout dev -- Gemfile.lock
$ git commit -m "Remove Gemfile.lock file from PR"
$ git push origin model
```